src/libraries/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_Compile.cs

```diff
@@ -149,5 +151,866 @@ public void TFS_470021()
+    <xs:simpleType name='foo'>
+        <xs:restriction base='xs:integer'>
+            <xs:minExclusive value='2' fixed='true'/>
+        </xs:restriction>
+    </xs:simpleType>
+    <xs:simpleType name='bar'>
+        <xs:restriction base='foo'>
+            <xs:minExclusive value='3'/>
+        </xs:restriction>
+    </xs:simpleType>
+</xs:schema>
+"
+                    },
+                    new object[]
+                    {  // minExclusive, derived type has smaller value
+                        @"<?xml version='1.0' encoding='utf-8' ?>
+<xs:schema elementFormDefault='qualified'
+           xmlns:xs='http://www.w3.org/2001/XMLSchema'>
+    <xs:simpleType name='foo'>
+        <xs:restriction base='xs:integer'>
+            <xs:minExclusive value='2' fixed='true'/>
+        </xs:restriction>
+    </xs:simpleType>
+    <xs:simpleType name='bar'>
+        <xs:restriction base='foo'>
+            <xs:minExclusive value='1'/>
+        </xs:restriction>
+    </xs:simpleType>
+</xs:schema>
+"
+                    },
+                    new object[]
+                    {  // minInclusive, derived type has larger value
+                        @"<?xml version='1.0' encoding='utf-8' ?>
+<xs:schema elementFormDefault='qualified'
+           xmlns:xs='http://www.w3.org/2001/XMLSchema'>
+    <xs:simpleType name='foo'>
+        <xs:restriction base='xs:integer'>
+            <xs:minInclusive value='3' fixed='true'/>
+        </xs:restriction>
+    </xs:simpleType>
+    <xs:simpleType name='bar'>
+        <xs:restriction base='foo'>
+            <xs:minInclusive value='4'/>
+        </xs:restriction>
+    </xs:simpleType>
+</xs:schema>
+"
+                    },
+                    new object[]
+                    {  // minInclusive, derived type has smaller value
+                        @"<?xml version='1.0' encoding='utf-8' ?>
+<xs:schema elementFormDefault='qualified'
+           xmlns:xs='http://www.w3.org/2001/XMLSchema'>
+    <xs:simpleType name='foo'>
+        <xs:restriction base='xs:integer'>
+            <xs:minInclusive value='3' fixed='true'/>
+        </xs:restriction>
+    </xs:simpleType>
+    <xs:simpleType name='bar'>
+        <xs:restriction base='foo'>
+            <xs:minInclusive value='2'/>
+        </xs:restriction>
+    </xs:simpleType>
+</xs:schema>
+"
+                    }
+                    // NOTE: the cases of totalDigits, with larger and smaller
+                    // values are not tested here as Issue 34426 was found
+                    // and will be addressed separately.
+                    // NOTE: the case of fractionDigits with a larger
+                    //   value is not tested here as Issue 34413 was found,
+                    //   and will be addressed separately.
+                    // NOTE: the case of fractionDigits with a smaller
+                    //   value is not tested here as Issue 34418 was found,
+                    //   and will be addressed separately.
```

Comment:
👍 @mrj001 thank you for finding these issues and reporting them, as we have an issues tracking them we don't need to keep these comments, even after those issues resolved these comments might forgotten, so please remove them

Reply:
Comments have been removed.